modules/release-notes/ref-release-notes-compatibility-matrix.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: REFERENCE
+[id="compatibility-matrix"]
+= Compatibility matrix
+
+The following table lists products and their versions compatible with {product-short} {product-version}:
+
+.Platform Compatibility Matrix
+[cols="1,2,1", options="header"]
+|===
+| {product-short} | Platform | {product-custom-resource-type} Version
+
+| `{product-version}`
+|
+- {ocp-brand-name} `{ocp-version-min}`-`{ocp-version}`
+
+- {eks-brand-name} ({eks-short})
+
+- {aks-brand-name} ({aks-short})
+
+- {aro-brand-name} ({aro-short})
+
+- {rosa-brand-name} ({rosa-short})
+
+- {gke-brand-name} ({gke-short})
+| `{product-backstage-version}`
+| `1.3`
+|
+- {ocp-brand-name} `4.14`-`4.16`
```

Comment:
Please use attributes for OCP min/max values: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/release-1.4/artifacts/attributes.adoc#L42-L43 

```suggestion
- {ocp-brand-name} `{ocp-version-min}`-`{ocp-version}`
```